monkey/monkey_island/cc/services/authentication_service/flask_resources/register.py

```diff
@@ -40,6 +41,10 @@ def post(self):
         except Exception:
             return responses.make_response_to_invalid_request()
 
+        errors = self._validate_username_and_password(username, password)
+        if errors:
+            return responses.make_response_to_invalid_request(errors)
```

Comment:
Request is valid, either password or username is not. `responses.make_response_to_invalid_request` is meant for invalid requests

Reply:
Flask is using 400 for the same bit. If we want to stay consistent we will go with that. Also check out this [link](https://stackoverflow.com/questions/6123425/rest-response-code-for-invalid-data). 

The only issue is the response which we get when we have invalid request and flask response.

One is `{"message": []}` other is `{"response": "errors": []}`